lib/grep.sh

```diff
@@ -21,7 +21,12 @@ elif grep_flag_available --exclude=.cvs; then
 fi
 
 # export grep settings
-alias grep="grep $GREP_OPTIONS"
+if [ -z ${GREP_OPTIONS+x} ]; then
+    alias grep="grep $GREP_OPTIONS"
+else
+    # no-op
+    :
+fi
```

Comment:
Have you tested your PR in your environment? I think the condition is inverted, and also there is a quoting issue. The above new code defines the alias when the variable `GREP_OPTIONS` is defined but has an empty value, and won't define the alias when `GREP_OPTIONS` has actual contents.

```suggestion
if [[ $GREP_OPTIONS ]]; then
    alias grep="grep $GREP_OPTIONS"
fi
```

Or you may actually always define the alias but with single quotes:

```suggestion
alias grep='grep $GREP_OPTIONS'
```

I think this second version should be the right fix because the first version has another problem that the shell metacharacters in `$GREP_OPTIONS` would be interpreted in the first version and can change the syntax structure of the command.

Reply:
Yes, you're right. Just changing to using single-quotes works fine. I used double-quotes just because that was what was there, originally. Please feel free to disregard this pull request, and make your changes directly. I'm a newbie at git/github, so I'm thrashing a bit.